main.go

```diff
@@ -1024,17 +1024,24 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
 		}
 	}
 
-	// set the correct integraton branches if we aren't performing a pull request again integration
+	// set the correct integration branches if we aren't performing a pull request again integration
```

Comment:
Since we're on the topic:
```suggestion
	// set the correct integration branches if we aren't performing a pull request against integration
```